sys/net/include/ieee802154_frame.h

```diff
@@ -77,19 +113,11 @@ typedef enum __attribute__((packed)) {
  */
 #define LETOHS(a)   HTOLES(a)
```

Comment:
This is a TODO: I think all four macros could be kicked out, just didn't want to touch them for this PR...